CMakeLists.txt

```
@@ -0,0 +1,42 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+cmake_minimum_required(VERSION 3.9.0)
+
+cmake_policy(SET CMP0048 NEW)
+
+project(Azure-Kinect-Samples LANGUAGES C CXX
+    VERSION 1.4)
+
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_C_STANDARD 99)
+
+set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
+set(CMAKE_LIBRARY_OUTPUT_DIRECTORY ${PROJECT_BINARY_DIR}/bin)
+
+FIND_PACKAGE(k4a REQUIRED)
+FIND_PACKAGE(k4abt REQUIRED)
+
+# These specific settings tell the loader to search the directory of the
+# executable for shared objects. This is done on Linux to emulate the default
+# behavior of the Windows loader, which searches for DLLs in the path of the
+# executable.
+if ("${CMAKE_SYSTEM_NAME}" STREQUAL "Linux")
+    set(CMAKE_BUILD_WITH_INSTALL_RPATH YES)
+    set(CMAKE_INSTALL_RPATH "\$ORIGIN")

```

We probably want to make the same change here I made for the k4a sdk.

```suggestion
    set(CMAKE_BUILD_RPATH "\$ORIGIN")
```
And remove the `set(CMAKE_BUILD_WITH_INSTALL_RPATH YES)`. This is because we only really need this for running the binary from the build directory.